src/main/include/RobotMap.h

```diff
@@ -142,4 +143,23 @@ struct RobotMap {
     //}
   };
   SwerveBase swerveBase;
-};
+
+  struct Climber {
+    // rev::CANSparkMax magMotor{99, rev::CANSparkMax::MotorType::kBrushless};
+    // wom::VoltageController magMotorGroup = wom::VoltageController::Group(magMotor);
+    // wom::CANSparkMaxEncoder magEncoder{&magMotor, 42};
+  
+    wom::Gearbox climberGearbox {
```

Comment:
fill this with stuff, you can assume you would be using a CanSparkMax controlling a NEO motor